src/main/java/com/security/security/config/SecurityConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.security.security.config;
+
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig {
+    @Bean
+    public UserDetailsService users() {
```

Comment:
Please provide a proper name for the created bean. By reading the bean name should be clear what is the scope of the given instance.